### PR TITLE
[chore] Update Dependencies for java and Java Script sample apps

### PR DESCRIPTION
**Pull request by vasireddy99**
Description:

Updates Java sample app (auto) and JavaScript sample app dependencies - downgrading the node-fetch to use 2.x [because of the breaking changes](https://github.com/node-fetch/node-fetch/blob/main/docs/v3-UPGRADE-GUIDE.md#breaking-changes). version 2 can still be used as the bug fixes will continue to be published for v2

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/integration-test.yml [hunk 1]
```diff
@@ -5,6 +5,10 @@ on:
     branches: [ master ]
   workflow_dispatch:
 
+concurrency:
+  group: integration-test-${{ github.ref_name }}
+  cancel-in-progress: true
```
Comment (bryan-aguilar) [inline .github/workflows/integration-test.yml:10]: Why is this required?

Comment (vasireddy99) [reply .github/workflows/integration-test.yml:10]: yeah, since the workflows are segregated per sample app and makes it little easy to troubleshoot. instead of running multiple runs at the same time given PRs are merged at the same time. Thought we could use this change

**Comment by vasireddy99** [comment]
> Can you also add a dependabot ignore for `node-fetch` major version?

Great idea, added it

**Review by humivo — APPROVED** [review]
